src/lib/components/02-organisms/CategorizedListRoot/CategorizedList/index.js

```diff
@@ -760,6 +752,8 @@ const CategorizedList = React.memo(
                   showCategoryTitles={props.showCategoryTitles}
                   onChangesUpdate={props.onChangesUpdate}
                   removeChangeObject={props.removeChangeObject}
+                  showValidationErrors={showValidationErrors}
+                  requiredMessage={props.requiredMessage}
```

Comment:
AFAIK requiredMessage isn't CategorizedList level property. Every component that is used by the CategorizedList have their own requiredMessage property. That's why requiredMessage={props.requiredMessage} should be removed.

Reply:
Yes it does need it.